SpatialGDK/Source/SpatialGDK/Public/Schema/Interest.h

```diff
@@ -7,6 +7,7 @@
 namespace SpatialGDK
 {
 using EdgeLength = Coordinates;
+using ResultType = TArray<Worker_ComponentId>;
```

Comment:
Didn't this already go in?

Reply:
I think it got lost in one of the rebase races